packages/components/src/components/Button/IconButton.tsx

```diff
@@ -5,6 +5,7 @@ import { EDSStyleSheet } from "../../styling";
 import { getBackgroundColorForButton } from "../../utils/getBackgroundColorForButton";
 import { Icon, IconName } from "../Icon";
 import { PressableHighlight } from "../PressableHighlight";
+import { DotProgress } from "../ProgressIndicator";
```

Comment:
In EDS, busy icon buttons use the CircularProgress component to indicate its state instead of dots. Maybe change to using this? 

<img width="507" alt="Screenshot 2024-10-24 at 12 09 19" src="https://github.com/user-attachments/assets/9a8ab057-f3cc-4544-8350-b53587164d86">